modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -1941,12 +1942,13 @@ stereoRectifyUncalibrated to compute the rectification transformation. :
 CV_EXPORTS_W Mat findFundamentalMat( InputArray points1, InputArray points2,
                                      int method = FM_RANSAC,
                                      double ransacReprojThreshold = 3., double confidence = 0.99,
-                                     OutputArray mask = noArray() );
+                                     OutputArray mask = noArray(), int maxIters = 1000 );
```

Comment:
This change violates OpenCV parameters order policy:
- input_arrays
- output_arrays (or function return if single output)
- algo_parameters
- optional_algo_parameters_with_default_values
- optional_output_arrays
- optional_other_output